static/app/actionCreators/pageFilters.tsx

```diff
@@ -334,7 +329,8 @@ export function updateDateTime(
   router?: Router,
   options?: Options
 ) {
-  PageFiltersActions.updateDateTime(datetime);
+  const {selection} = PageFiltersStore.getState();
+  PageFiltersStore.updateDateTime({...selection.datetime, ...datetime});
```

Comment:
This is somewhat signficant. The types were wrong so it was possible we were updating the `datetime` value in the store with a partial date time.

I don't... think this will break anything.